src/tools/tidy/src/style.rs

```diff
@@ -141,17 +141,27 @@ pub fn check(path: &Path, bad: &mut bool) {
     super::walk(path, &mut super::filter_dirs, &mut |entry, contents| {
         let file = entry.path();
         let filename = file.file_name().unwrap().to_string_lossy();
-        let extensions = [".rs", ".py", ".js", ".sh", ".c", ".cpp", ".h"];
+        let extensions = [".rs", ".py", ".js", ".sh", ".c", ".cpp", ".h", ".md"];
         if extensions.iter().all(|e| !filename.ends_with(e)) ||
            filename.starts_with(".#") {
             return
         }
 
+        if filename.ends_with(".md") &&
+           file.parent()
+               .unwrap()
+               .file_name()
+               .unwrap()
+               .to_string_lossy() != "error_codes" {
+            // Only error codes md files are checked.
+            return;
+        }
+
         if contents.is_empty() {
             tidy_error!(bad, "{}: empty file", file.display());
         }
 
-        let max_columns = if filename == "error_codes.rs" {
+        let max_columns = if filename == "error_codes.rs" || filename.ends_with(".md") {
```

Comment:
This is pointless, right? `error_codes.rs` would never reach here anyway

Reply:
Why? From what I can read from the code, it can definitely reach here.

Reply:
Ah, I misunderstood. But IIRC we don't expect error_codes.rs to contain error code descriptions anymore, so presumably it doesn't need a separate column count?

Reply:
Well, it doesn't hurt to keep it at this point because it's not supposed to have content anyway.